samples/wifi/shell/README.rst

```diff
@@ -0,0 +1,104 @@
+.. _wifi_sample:
+
+Wi-Fi: nRF chipsets
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample allows testing Nordic Wi-Fi chipsets.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf7002dk_nrf5340_cpuapp, nrf7002dk_nrf5340_cpuapp_ns, nrf7002dk_nrf5340_cpunet
+
+
+Overview
+********
+
+You can use this sample as a starting point to experiment and test Nordic Wi-Fi chipsets.
+
+The sample can be used to perform all Wi-Fi operations in 2.4/5/6GHz bands depending on the capabilities supported in the underlying
+chipset. You can associate and ping to any Wi-Fi capable access point in STA mode. It can also support other Wi-Fi modes of operation
+depending on the underlying chipset capabilities.
```

Comment:
```suggestion
The sample can be used to perform all Wi-Fi operations in 2.4/5/6GHz bands depending on the capabilities supported in the underlying chipset.
You can associate and ping to any Wi-Fi capable access point in STA mode.
It can also support other Wi-Fi modes of operation depending on the underlying chipset capabilities.
```
Remember, each sentence on a separate line. Can we open the abbreviation STA (and perhaps add it to the glossary)?